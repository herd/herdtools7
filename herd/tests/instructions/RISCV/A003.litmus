RISCV A003

(* Test SRLI Instruction*)

{ 0:a0 = 4}

P0;
srli a0, a0, 1;

forall 0:a0 = 2
