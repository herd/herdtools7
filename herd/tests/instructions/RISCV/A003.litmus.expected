Test A003 Required
States 1
0:x10=2;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:x10=2)
Observation A003 Always 1 0
Hash=0f76c62e95e0af150e8776b7b0859e14

